src/types.ts

```diff
@@ -0,0 +1,152 @@
+type SimpleMalopFeature = {
+  totalValues: number;
+  values: Array<string> | null;
+};
+
+type ElementMalopFeature = {
+  totalValues: number;
+  elementValues: Array<{
+    elementType: string;
+    guid: string;
+    name: string;
+    hasSuspicions: boolean;
+    hasMalops: boolean;
+  }>;
+  totalSuspicious: number;
+  totalMalicious: number;
+};
+
+export type Malop = {
+  simpleValues: {
+    hasRansomwareSuspendedProcesses: SimpleMalopFeature;
+    decisionFeature: SimpleMalopFeature;
+    rootCauseElementCompanyProduct: SimpleMalopFeature;
+    malopStartTime: SimpleMalopFeature;
+    detectionType: SimpleMalopFeature;
+    malopActivityTypes: SimpleMalopFeature;
+    elementDisplayName: SimpleMalopFeature;
+    creationTime: SimpleMalopFeature;
+    isBlocked: SimpleMalopFeature;
+    rootCauseElementTypes: SimpleMalopFeature;
+    rootCauseElementNames: SimpleMalopFeature;
+    malopLastUpdateTime: SimpleMalopFeature;
+    allRansomwareProcessesSuspended: SimpleMalopFeature;
+    rootCauseElementHashes: SimpleMalopFeature;
+    managementStatus: SimpleMalopFeature;
+    closeTime: SimpleMalopFeature;
+    closerName: SimpleMalopFeature;
+    customClassification: SimpleMalopFeature;
+  };
+  elementValues: {
+    affectedUsers: ElementMalopFeature;
+    affectedMachines: ElementMalopFeature;
+  };
+  suspicions: null; // Unclear
+  filterData: {
+    sortInGroupValue: string;
```

Comment:
Have added // Unclear wherever the type of the respective field was not clear from the API. (In most cases, these fields were marked as null.)